script/generate_artifact.sh

```diff
@@ -2,4 +2,5 @@
 mkdir bin
 cp target/release/{plume,plm} bin
 strip -s bin/*
-tar -cvzf plume.tar.gz bin/ static/ migrations/$FEATURES
+tar -cvzf plume.tar.gz bin/ static/
```

Comment:
this one should be with `migrations/$FEATURES`, and without `static/`

Reply:
migrations don't require to be shipped as they are integrated into `plm`, and I think it should be with `static/` so that for x86_64 users, there is only a single archive to download, un-tgz, and run